pkg/ccl/backupccl/backuppb/backup.proto

```diff
@@ -131,7 +131,16 @@ message BackupManifest {
   int32 descriptor_coverage = 22 [
     (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/sem/tree.DescriptorCoverage"];
 
-  // NEXT ID: 27
+  // IsSlimManifest is set to true if the backup manifest has some of its alloc
+  // heavy repeated fields nil'ed out and written as supporting SST files
+  // instead. Currently, the fields that are nil'ed out are:
+  // - Files
+  //
+  // TODO(adityamaru): Delete in 23.2 since all backups in 23.1+ will write slim
```

Comment:
nit: "delete when backwards compatibility with 22.2 is dropped" -- don't assume which version that'll be in.

Reply:
old habits die hard, changed